db/1.14/modules/ROOT/pages/database-connector-troubleshooting.adoc

```diff
@@ -0,0 +1,303 @@
+
+[source,xml,linenums]
+----
+	<Line 43, Column 57>: XML-24509: (Error) Duplicated definition for: 'identifiedType' <Line 60, Column 28>: XML-24509: (Error) Duplicated definition for: 'beans' <Line 140, Column 34>: XML-24509: (Error) Duplicated definition for: 'description'
+----
+To deal with this issue, do either of the following:
+
+* Add the `xmlparserv2.jar` file as an additional plugin dependency in the Database Connector, so the dependency is visible only from within the connector. +
+Though, that won't work if you need to share an Oracle connection among modules, which is the case if you want to provide a DataSource as a Spring configured bean.
+* Disable the XML parser SPI lookup by setting the following system properties, so Mule internal components always get the XML parser they expect:
+
+.System Properties:
+[source,xml,linenums]
+----
+-Djavax.xml.parsers.SAXParserFactory=com.sun.org.apache.xerces.internal.jaxp.SAXParserFactoryImpl
+-Djavax.xml.parsers.DocumentBuilderFactory=com.sun.org.apache.xerces.internal.jaxp.DocumentBuilderFactoryImpl
+----
+
+These properties avoid any trouble if, for instance, you need to set up Oracle .jars as shared libraries.
+
+For more information about how to set system properties, refer to the xref:mule-runtime::mule-app-properties-system.adoc[system properties] documentation.
+
+
+== Troubleshoot Stored Procedure Operation With Named Parameters
+
+When using the *Stored procedure* operation and the named parameters binding feature, some exceptions are expected in certain scenarios.
+
+The following stored procedure definition is used for each scenario:
+
+[source,xml,linenums]
+---
+	CREATE PROCEDURE SP_DEMO(PARAMETERA VARCHAR2, PARAMETERB VARCHAR2, PARAMETERC NUMBER)
+		IS
+		BEGIN
+			INSERT INTO TABLE_DEMO_DEFAULT_VALUE (PARAMETER_A, PARAMETER_B, PARAMETER_C)
+			VALUES (PARAMETERA, PARAMETERB, PARAMETERC);
+		END;
+---
+
+=== Scenario: Mixing Named and Ordinal Parameter Bindings
+
+Suppose the following stored procedure execution that uses both named and ordinal parameter bindings:
+
+[source,xml,linenums]
+----
+	<db:stored-procedure config-ref="Database_Config">
+		<db:sql><![CDATA[call SP_DEMO(PARAMETERB => :parameterb, PARAMETERC=> :parameterc, :parametera)]]></db:sql>
+		<db:input-parameters><![CDATA[#[{parametera: 'HELLO', parameterb: 'A', parameterc: 123456}]]]></db:input-parameters>
+	</db:stored-procedure>
+----
+
+The following exception is expected:
+
+[source,xml,linenums]
+----
+	Root Exception stack trace:
+	org.mule.runtime.api.exception.MuleRuntimeException: Could not resolve query: call SP_DEMO(PARAMETERB => :parameterb, PARAMETERC=> :parameterc, :parametera).Error: org.mule.db.commons.internal.parser.QueryTemplateParsingException: Invalid Stored Procedure expression: Ordinal binding and Named binding cannot be combined
+----
+
```

Comment:
I would remove "Scenario:" here and elsewhere. I don't think it's necessary and it makes things harder to skim.